enterprise/kernel/src/test/java/org/neo4j/kernel/impl/enterprise/lock/forseti/ForsetiFalseDeadlockTest.java

```diff
@@ -293,6 +293,33 @@ public void testMildlyForFalseDeadlocks() throws Exception
 
     private void loopRunTest( int testRuns ) throws InterruptedException, java.util.concurrent.ExecutionException
     {
+        List<Throwable> exceptionList = new ArrayList<>();
+        loopRun( testRuns, exceptionList );
+
+        if ( !exceptionList.isEmpty() )
+        {
+            // We saw exceptions. Run it 99 more times, and then verify that our false deadlock rate is less than 2%.
+            int additionalRuns = testRuns * 99;
+            loopRun( additionalRuns, exceptionList );
+            double totalRuns = additionalRuns + testRuns;
+            double failures = exceptionList.size();
+            double failureRate = failures / totalRuns;
+            if ( failureRate > 0.2 )
```

Comment:
should it be 0.02 instead? failure reporting threshold is more than 20%?

Reply:
Oh shoot, you're right!